client/lib/interval/README.md

```diff
@@ -1,43 +1,41 @@
 # Interval
 
-An interface into a global timer coalescing interval runner.
+A React-oriented, declarative interval runner.
 
-_**An interface**_ because this component handles registering and un-registering the given `onTick` action with the global runner.
-
-_**timer coalescing**_ because the global runner will run at the same time all of the actions registered for a given interval period. For example, if four actions are registered for running once per minute, they will all run at one point in time every minute instead of having four different run times, each time repeating every minute. This is better for things like battery life because it allows for longer and deeper periods of sleep between actions.
```

Comment:
This claim seems to be completely inaccurate. I'm not sure why the claim was made.

Reply:
https://github.com/Automattic/wp-calypso/commit/d68eef66197e571fb0fdf1561e5ee9cef7584479

This claim was accurate when it was made

Reply:
Thanks for providing detail @dmsnell 🙇